context/AppContextProvider.tsx

```diff
@@ -1,42 +1,26 @@
-"use client";
+'use client';
 
 import 'react-toastify/dist/ReactToastify.css';
-
 import { useTranslations } from 'next-intl';
-import {
-  createContext,
-  useState,
-  SetStateAction,
-  ReactNode,
-  useEffect
-} from 'react';
+import { createContext, useState, ReactNode, useEffect, SetStateAction } from 'react';
 import { toast } from 'react-toastify';
-
 import { Pi } from '@pinetwork-js/sdk';
-import axiosClient, {setAuthToken} from '@/config/client';
-import { onIncompletePaymentFound } from '@/util/auth';
+import axiosClient, { setAuthToken } from '@/config/client';
+import { autoSigninUser, onIncompletePaymentFound, PiAuthentication } from '@/util/auth';
```

Comment:
I don't believe we're referencing `PiAuthentication` anywhere.  This can be removed.